boards/arm/verdin_imx8mp_m7/verdin_imx8mp_m7_ddr.yaml

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2023 Toradex
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+identifier: verdin_imx8mp_m7_ddr
+name: Toradex Verdin iMX8M Plus (DDR)
+type: mcu
+arch: arm
+ram: 2048
+flash: 2048
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+testing:
+  ignore_tags:
```

Comment:
Question here- do we need `ignore_tags` set in order to pass CI on this board? Otherwise, I'd like to avoid defining it.

Reply:
I tested locally and it seems to not be necessary. I'll remove the `ignore_tags`.